x-pack/qa/security-example-spi-extension/src/main/java/org/elasticsearch/example/SpiExtensionPlugin.java

```diff
@@ -22,8 +23,10 @@
 public class SpiExtensionPlugin extends Plugin implements ActionPlugin {
 
     @Override
-    public Collection<String> getRestHeaders() {
-        return Arrays.asList(CustomRealm.USER_HEADER, CustomRealm.PW_HEADER);
+    public Collection<RestHeaderDefinition> getRestHeaders() {
+        return Arrays.asList(
+            new RestHeaderDefinition(CustomRealm.USER_HEADER, true),
+            new RestHeaderDefinition(CustomRealm.PW_HEADER, true));
```

Comment:
Shouldn't these be `false` ?
I know it doesn't matter for testing purposes, but we point people to this as an example, and I think we want the default expaction for security examples to be single-valued headers.